go.mod

```diff
@@ -1,9 +1,28 @@
 module github.com/ChainSafe/chaindb
 
-go 1.15
+go 1.19
 
 require (
 	github.com/ChainSafe/log15 v1.0.0
 	github.com/dgraph-io/badger/v2 v2.0.3
 	github.com/stretchr/testify v1.4.0
 )
+
+require (
+	github.com/DataDog/zstd v1.4.1 // indirect
+	github.com/cespare/xxhash v1.1.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/dgraph-io/ristretto v0.0.2-0.20200115201040-8f368f2f2ab3 // indirect
+	github.com/dgryski/go-farm v0.0.0-20190423205320-6a90982ecee2 // indirect
+	github.com/dustin/go-humanize v1.0.0 // indirect
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/golang/protobuf v1.3.1 // indirect
+	github.com/golang/snappy v0.0.1 // indirect
+	github.com/mattn/go-colorable v0.1.2 // indirect
+	github.com/mattn/go-isatty v0.0.8 // indirect
+	github.com/pkg/errors v0.8.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859 // indirect
+	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb // indirect
+	gopkg.in/yaml.v2 v2.2.2 // indirect
+)
```

Comment:
why the big diff?

Reply:
Those deps are needed in order to upgrade to go1.19 but are not listed in the go mod of the direct deps, this is what I understand from here https://stackoverflow.com/questions/61115111/how-to-avoid-indirect-dependencies-in-go-mod-file, and we can get a dependency graph by executing `go mod graph` and see the relationship